app/Http/Controllers/Backoffice/Security/Modules/ModulesGetController.php

```
@@ -0,0 +1,137 @@
+            if (empty($contextId)) {
+                return new JsonResponse(
+                    [
+                        'title' => 'Bad Request',
+                        'status' => Response::HTTP_BAD_REQUEST,
+                        'detail' => 'Context ID is required'
+                    ],
+                    Response::HTTP_BAD_REQUEST
+                );
+            }
+
+            $response = ($this->searcher)(new ModulesSearcherRequest($contextId));
```

PHPStan (2.0.3)

Parameter #1 $contextId of class MedineTech\Backoffice\Security\Modules\Application\Search\ModulesSearcherRequest constructor expects string, array|string given.


(argument.type)